File: taskrouter/v1/workspace/create-example-1/create-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Taskrouter.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var workspace = WorkspaceResource.Create(
            eventCallbackUrl: new Uri("http://requestb.in/vh9reovh"),
            template: "FIFO",
            friendlyName: "NewWorkspace"
        );

        Console.WriteLine(workspace.Sid);
    }
}

File: taskrouter/v1/workspace/create-example-1/create-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

workspace = @client.taskrouter
                   .workspaces
                   .create(
                      event_callback_url: 'http://requestb.in/vh9reovh',
                      template: 'FIFO',
                      friendly_name: 'NewWorkspace'
                    )

puts workspace.sid
